tests/foreman/api/test_syncplan.py

```diff
@@ -371,21 +371,21 @@ def test_positive_update_interval(self):
         :CaseImportance: Critical
         """
         for interval in valid_sync_interval():
-            with self.subTest(interval):
-                sync_plan = entities.SyncPlan(organization=self.org)
-                result = sync_plan.get_fields()['interval']
-                sync_plan.interval = sample(
-                    set(result.choices) - set([interval]),
-                    1
-                )[0]
-                sync_plan = sync_plan.create()
-                sync_plan.interval = interval
-                if (interval == 'custom cron'):
-                    sync_plan.cron_expression = gen_choice(valid_cron_expressions())
-                    sync_plan = sync_plan.update(['interval', 'cron_expression'])
-                else:
-                    sync_plan = sync_plan.update(['interval'])
-                self.assertEqual(sync_plan.interval, interval)
+            sync_plan = entities.SyncPlan(
+                description=gen_string('alpha'),
+                organization=self.org,
+                interval=interval
+            )
+            if interval == SYNC_INTERVAL['custom']:
+                sync_plan.cron_expression = gen_choice((valid_cron_expressions()))
+            sync_plan = sync_plan.create()
+
+            if (interval == 'custom cron'):
```

Comment:
I know this is unchanged from old version, but please use `interval == SYNC_INTERVAL['custom']` and drop parentheses here.

Reply:
OK